### DISCUSSION TITLE
Show Effect for Mega Menu

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21218] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

Before the release of Megamanu, I wanted to suggest a great and important feature for Megamanu. Please add **Show Effect** feature for megamenu. This feature causes the megamenu to appear with the effect. Also the exit effect would be great for it.

### Describe the solution you'd like

In the video below, Mega menu appears as a slider up.

https://user-images.githubusercontent.com/88034902/216111589-f8cff416-40da-415c-b2b5-1178cb6ddb94.mp4

Elementor team, please keep this feature in mind. It will be very attractive and great.❤

**Comment by toineenzo** [comment]
You could do this by adding an animation for the container for the certain mega menu item

**Reply by Michaelgimii** [reply]
Are you sure this works? I tested this in the nested tabs widget, which is a similar widget to Mega Menu. It works, but it doesn't look cool.
If you choose slide in up animation, a scroll bar for the tab widget will appear when the animation is running. I hope this does not happen in Mega Manu.
